### ISSUE TITLE
Tab-dependent layer visibility

### ISSUE THREAD

**Issue by leebock** [Esri/shortlist-storytelling-template-js#18]
An enhancement that would be good in the Shortlist is the ability for the author to have different supporting layers (i.e. points, lines and polys that aren't used to drive the tabs) appear on the map according to which tab is chosen. So for example, for on a 'places to go' shortlist, I could have a 'Trails' tab and when the user goes to that tab they'd see the numbered points as usual but they'd also see the trails as lines I've added. If they switch to a different tab those trail lines wouldn't be displayed. At the moment in the Shortlist, your supporting features are always on the map, so if you added trails as lines, they would be displayed all the time. So the enhancement would let you hide and display different supporting features depending on which tab is selected. This would be pretty cool. A 'Parks' tab could show people the park boundaries, a 'Transport' tab could show people the rail network and bus stops, etc without cluttering the map up.

**Comment by Nataliya1** [comment]
Yes, this would be great. Looks like it's been marked as "enhancement", any ideas when it would be available?

**Comment by cooney** [comment]
Hi @Nataliya1, thanks for your interest in Shortlist.  This is certainly on our radar, didn't make our latest round of development, but is something we would like to include in the next couple of releases.  We will let you know about our progress.

**Comment by Nataliya1** [comment]
I was just curious if this idea has made it to the development yet. Thank you.

**Comment by cooney** [comment]
It has not.  We are currently working on version 2.0 of shortlist that will be integrated and hosted with AGOL, and like other Story Maps templates, will have an interactive builder.  It may make that round of development.  In the mean time, check out this post on GeoNet for information on possibly customizing for this behavior yourself: https://geonet.esri.com/message/557885#557885

**Comment by Nataliya1** [comment]
@cooney that is exactly what I need. I will give it a try. I also like that the shortlist 2.0 will be integrated and hosted with AGOL. Thank you.
